types/grecaptcha/index.d.ts

```diff
@@ -46,7 +46,7 @@ declare namespace ReCaptchaV2 {
     /**
      * Your sitekey.
      */
-    sitekey: string;
+    sitekey?: string;
```

Comment:
The `Parameters` interface describes the object required by the `grecaptcha.render` method.
According to the docs, the `sitekey` property is required.
The `data-*` attributes are used only on the programmatically mode, which is not described by this definition.